docs/upgrade.html

```diff
@@ -66,6 +66,10 @@ <h5><a id="upgrade_110_notable" href="#upgrade_110_notable">Notable changes in 1
     <li>The kafka artifact in Maven no longer depends on log4j or slf4j-log4j12. Similarly to the kafka-clients artifact, users
         can now choose the logging back-end by including the appropriate slf4j module (slf4j-log4j12, logback, etc.). The release
         tarball still includes log4j and slf4j-log4j12.</li>
+    <li>Kafka Streams is more robust against broker communication errors. Instead of stopping the Kafka Streams client with a fatal exception,
+	Kafka Streams tries to self-heal and reconnect to the cluster. Using the new <code>AdminClient</code> you have also more control how often
+	Kafka Streams should retry and configure timeouts fine-grained (instead of hard coded retries as in older version).</li>
+    <li>Kafka Streams rebalance time was reduced further making Kafka Streams more responsive.</li>
```

Comment:
nit: could we add a reference link to `doc/streams/upgrade.html` for users to read how to set the configs etc?